packages/create-calypso-config/src/index.ts

```diff
@@ -71,7 +71,7 @@ const isEnabled =
 	( feature: string ): boolean => {
 		// Feature flags activated from environment variables.
 		if (
-			process.env.ACTIVE_FEATURE_FLAGS &&
+			process?.env?.ACTIVE_FEATURE_FLAGS &&
```

Comment:
This would result in a reference error because `process` is also not defined in the browser

I tested to confirm this 
<img width="1728" alt="console" src="https://github.com/Automattic/wp-calypso/assets/9039613/a142ed31-56c9-4454-9bf3-a8cfb46d335e">
<img width="556" alt="source" src="https://github.com/Automattic/wp-calypso/assets/9039613/887b8ad3-32bc-4cf0-bc53-98375bce5203">

We can probably do something in line of this example or similar:

```ts
const env = typeof process !== 'undefined' ? process.env : {};
if (env?.ACTIVE_FEATURE_FLAGS &&....)
```

Reply:
> We can probably do something in line of this example or similar:
> ```
> const env = typeof process !== 'undefined' ? process.env : {};
> if (env?.ACTIVE_FEATURE_FLAGS &&....)
> ```

Just noting that the above seems to be fixing the issue well. 

[Related comment](https://github.com/Automattic/wp-calypso/pull/91908/files#r1650737676)

Reply:
Hi folks!

Thank you for checking and reporting the issues!

I fixed it here: https://github.com/Automattic/wp-calypso/pull/92029/commits/eaa76574f81829e4c52843b7edcbb1daf6e34eea

I fixed it differently but with the same result. The reason is the way that these variables are parsed during the build. 😉

Please, let me know if that works or if you face any other issues.

Reply:
Nice! I can confirm this fixes the issue in my test.

I think we could even consider a simpler check - its "middle" part (but I am approving this PR already if you would like to merge it as is):

```suggestion
			process.env &&
```

Reply:
Thank you for the suggestion @ivan-ottinger!

I'll skip change because of https://github.com/mrsteele/dotenv-webpack?tab=readme-ov-file#processenv-stubbing--replacing

More details: https://github.com/mrsteele/dotenv-webpack/blob/master/src/index.js#L150-L157

Basically, we have this weird result with the `"MISSING_ENV_VAR"` when transpiling through webpack and using `process.env` only:

<img width="560" alt="Screenshot 2024-06-24 at 14 11 47" src="https://github.com/Automattic/wp-calypso/assets/876340/2d27f940-cd32-4edf-b36e-e9af3c72fac7">

Reply:
Thanks for sharing, Renatho. That's good to know.